lib/oli_web/pow/user_context.ex

```diff
@@ -60,37 +61,58 @@ defmodule OliWeb.Pow.UserContext do
   """
   @impl true
   def create(params) do
-    params =
-      with %{"section" => section_slug} <- params,
-           %Section{skip_email_verification: true} <- Sections.get_section_by_slug(section_slug) do
-        email_confirmed_at = DateTime.truncate(DateTime.utc_now(), :second)
-        Map.put(params, "email_confirmed_at", email_confirmed_at)
-      else
-        _ -> params
-      end
-
-    %User{}
-    |> User.verification_changeset(params)
-    |> Repo.insert()
-    |> case do
-      {:ok, user} ->
-        if Application.fetch_env!(:oli, :age_verification)[:is_enabled] == "true" do
-          Logger.info(
-            "User (id: #{user.id}, email: #{user.email}) created successfully with age verification"
-          )
-        end
+    case Accounts.get_user_by(%{email: params["email"]}) do
+      %User{email: email} = user ->
+        if user.email_confirmed_at,
```

Comment:
I think is a very similar logic than in the author context what about doing the same here?